src/parser.c

```diff
@@ -1260,6 +1260,22 @@ void read_lvalue(lvalue_t *lvalue,
     while (lex_peek(T_open_square, NULL) || lex_peek(T_arrow, NULL) ||
            lex_peek(T_dot, NULL)) {
         if (lex_accept(T_open_square)) {
+            /* if subscripted member's is not yet resolved, deference to resolve
```

Comment:
There should be a comma (`,`) appearing right after "e.g.". That is, `e.g.,`.

See https://www.scribbr.com/definitions/for-example-abbreviation/